pkg/logstash.sysv

```diff
@@ -114,6 +114,27 @@ force_stop() {
   fi
 }
 
+config_test() {
+  # Check if a config file exists
+  if [ ! "$(ls -A $LS_CONF_DIR/*.conf 2> /dev/null)" ]; then
```

Comment:
`LS_CONF_DIR` [defaults to a directory](https://github.com/Mister-X-/logstash/blob/master/pkg/logstash.sysv#L35), not a wildcard. Checking for `*.conf` here will falsely indicate that there are no configuration files if someone has a file named `$LS_CONF_DIR/fancy` (which Logstash normally would see as a config file)